### ISSUE TITLE
Shader naming too technical, needs human friendly names

### ISSUE THREAD

**Issue by fastrizwaan** [ErwanDouaille/gse-shader#2]
Hi,

The "Shader" name itself is not clear to non-techies,  "Color effects" or something which describes  the term in layman term would be great. Perhaps a tooltip would suffice.

Default - Reset
Luminosity - brightness
InvertColors - Negative

Thanks.

**Comment by ErwanDouaille** [comment]
Commit https://github.com/ErwanDouaille/gse-shader/commit/913f000e002d4fdb8518857f74223cab37ac3a9b
